assets/js/modules/analytics/components/module/ModuleOverviewWidget/Overview.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Overview component the ModuleOverviewWidget widget.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Grid, Row, Cell } from '../../../../../material-components';
+import DataBlock from '../../../../../components/DataBlock';
+import PreviewBlock from '../../../../../components/PreviewBlock';
+import { calculateChange } from '../../../../../util';
+
+export default function Overview( props ) {
+	const {
+		loaded,
+		users,
+		sessions,
+		bounce,
+		duration,
+		selectedStat,
+		handleStatSelection,
+	} = props;
+
+	if ( ! loaded ) {
+		return (
+			<PreviewBlock width="100%" height="190px" />
+		);
+	}
+
+	const getDatapointAndChange = ( [ report ], divider = 1 ) => ( {
+		datapoint: report?.data?.totals?.[0]?.values?.[0] / divider,
+		change: calculateChange(
+			report?.data?.totals?.[1]?.values?.[0],
+			report?.data?.totals?.[0]?.values?.[0],
+		),
+	} );
```

Comment:
Maybe move out of the component since it doesn't have to be inside of it regarding dependencies?

Reply:
Updated.